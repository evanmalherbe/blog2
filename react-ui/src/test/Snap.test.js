import React from "react";
import Button from "react-bootstrap/Button";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(<Button variant="primary">Facebook</Button>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
