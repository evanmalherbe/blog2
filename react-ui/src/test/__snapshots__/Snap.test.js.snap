// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="btn btn-primary"
  disabled={false}
  type="button"
>
  Facebook
</button>
`;
